### PR TITLE
added http handler

### PR DESCRIPTION
**Pull request by hedwigz**

### CODE REVIEW

File: entviz.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	_ "embed"
 	"encoding/json"
 	"html/template"
+	_ "net/http/pprof"
```
Comment (a8m) [inline entviz.go:8]: ??

File: README.md [hunk 2]
```diff
@@ -9,7 +9,16 @@ run
...
+your html will be saved at `ent/schema-viz.html`
+# server via http
+You can use the helper function `entviz.Serve` to easily serve the static html page over http
+```golang
+h, err := entviz.Serve("./ent/schema", &gen.Config{})
+if err != nil {
+  panic(err)
+}
+http.ListenAndServe("localhost:3002", h)
+```
```
Comment (a8m) [inline README.md:21]: Not sure this example is relevant after the new changes.

**Review by a8m — APPROVED** [review]

File: entviz.go (hunk 1 above)
Comment (hedwigz) [reply entviz.go:8]: whoops. nice catch

File: README.md (hunk 2 above)
Comment (hedwigz) [reply README.md:21]: correct, I accidentally put the fix in #4

**Pull request merged**